pybutton/resources/accounts.py

```diff
@@ -0,0 +1,78 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from .resource import Resource
+
+
+class Accounts(Resource):
+    '''Manages interacting with Button Orders with the Button API
+
+    Args:
+        api_key (string): Your organization's API key.  Do find yours at
+                https://app.usebutton.com/settings/organization.
+        config (dict): Configuration options for the client. Options include:
+            hostname: Defaults to api.usebutton.com.
+            port: Defaults to 443 if config.secure, else defaults to 80.
+            secure: Whether or not to use HTTPS. Defaults to True.
+            timeout: The time in seconds for network requests to abort.
+              Defaults to None.
+              (N.B: Button's API is only exposed through HTTPS. This option is
+              provided purely as a convenience for testing and development.)
+
+    Raises:
+        pybutton.ButtonClientError
+
+    '''
+
+    def all(self):
+        '''Get a list of available accounts
+
+        Raises:
+            pybutton.ButtonClientError
+
+        Returns:
+            (pybutton.Response) The API response
+
+        '''
+
+        return self.api_get('/v1/affiliation/accounts')
+
+    def transactions(self, account_id, cursor=None, start=None, end=None):
```

Comment:
May be worth spelling out in the docs what arguments are required and what are optional.